tools/importer-msgraph-metadata/pipeline/interface.go

```diff
@@ -0,0 +1,33 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+type RunInput struct {
+	Logger hclog.Logger
+
+	ApiVersions        []string
+	MetadataDirectory  string
+	OpenApiFilePattern string
+	OutputDirectory    string
+	ProviderPrefix     string
+	Tags               []string
+
+	// ModelsPerService will output models and constants for each service, duplicating where necessary. However, this
+	// yields >400,000 files at over 1.5GB, so setting this to false will instead output a single Models namespace
+	ModelsPerService bool
```

Comment:
I don't think so, and I've pulled it out since we do actually have a mix of common and service-specific models, so that's now handled with more nuance.